docs/design/storagenode-graceful-exit.md

```diff
@@ -0,0 +1,94 @@
+# Storage Node Graceful Exit Design Document
+
+
+## Overview
+
+When a Storage Node wants to leave the network but does not want to lose their escrow we need to have a mechanism for them to exit the network “gracefully”.
+
+This process including the Storage nodes transferring their pieces to other nodes so that the satellite does not have to repair those pieces because of a node exiting abruptly. The process of a Storage Node exiting gracefully including that node requesting a list of Storage Nodes to send their pieces to and updating the satellite with what nodes are now storing those pieces. Graceful Exit for Storage Nodes is beneficially for both the Storage Node and satellite because the storage node receives their escrow and the satellite saves money from not having to repair files.
+
+
+## Goals
+
+- Give Storage Nodes a mechanism to leave the network while receiving their escrows, and reduce repair caused by node churn on satellites. 
+
+
+## Non Goals
+
+- Sending the storage nodes escrows.
+	- The sending of tokens will happen through our normal token payment process.
+
+
+## Scenarios (MVP)
+
+- A storage node gracefully exits a satellite or all satellites.
+- A storage node fails to complete the graceful exit process.
+
+
+## Scenarios (Non MVP)
+
+- A storage node runs out of bandwidth during the graceful exit process.
+- A storage node wants to “partially” exit a satellite.
+- A storage node wants to rejoin a satellite it previously exited.
+
+
+## Business Requirements/ Job Stories (MVP)
+
+- When a Storage node no longer wants to store data for a satellite I want them to have the ability to run graceful exit for a specific satellite so that they do not lose their escrow on that satellite. 
```

Comment:
When these are separated into sections like:

```
### When Triggered
...

### When Finished
...

### When Exited Network
...
```

It'll be much clearer.

Reply:
updated with sections